### ISSUE TITLE
STRETCH - OPEN/CLOSED ORDER FILTERING

### ISSUE THREAD

**Issue by keanacobarde** [nss-evening-cohort-24/pos-system-the-ronis#59]
# User Story
I, as the user, should be able to select on some sort of button or drop down to toggle between open and closed orders. 

# Acceptance Criteria
AS PULLED FROM STRETCH GOALS
```
As an authenticated user, while looking at the list of orders, there should be an affordance (like a button or dropdown) for me to toggle between the open and closed orders. The open orders should show up first.
```
WHEN, the user selects on this button/dropdown, the function should get the orders. Once orders are obtained from Firebase through getOrders() API call, the response should be filtered into OPEN/CLOSED, depending on user input. Then, showCards() should take filtered array and display cards.

# Dependecies
MVP must be completed before the onset of this stretch goal. 

# Dev Notes
- Some sort of button or dropdown menu may need to be introduced to either the navbar, or a new section will need to be created for it within the the domBuilder. Or, whenever the orders.js page is rendered/startApp is rendered - the button/drop down rendering function will need to be called. 
- Utilize getCards() API call. Filter results based on state of button. Re-render based on filtered array.
